scripts/fetch_fnet_waveform.py

```
@@ -0,0 +1,1061 @@
+                   VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)""",
+                (
+                    rec["station_id"], rec["date_str"], rec["segment_hour"],
+                    rec["rms_z"], rec["rms_h"], rec["hv_ratio"],
+                    rec["lf_power"], rec["hf_power"], rec["spectral_slope"],
+                    rec.get("vlf_power"), rec.get("vlf_hv_ratio"),
+                    rec["n_samples"], rec["latitude"], rec["longitude"],
+                    rec["region"], now_str,
+                ),
+            )
+            inserted += 1
+        except Exception as exc:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)